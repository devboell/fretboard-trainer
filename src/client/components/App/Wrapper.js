import styled from 'styled-components'

const Wrapper = styled.div`
  width: 80%;
  margin: 10px 10%;
`

export default Wrapper
