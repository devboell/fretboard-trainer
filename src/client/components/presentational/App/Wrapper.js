import styled from 'styled-components'

const Wrapper = styled.div`
  width: 90%;
  margin: 0px 5%;
`

/** @component */
export default Wrapper
