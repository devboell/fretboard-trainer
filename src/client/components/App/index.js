import React from 'react'
import { Route } from 'react-router-dom'
import Nav from 'components/Nav'
import TrainingPage from 'components/TrainingPage'
import EditorPage from 'components/EditorPage'
import Wrapper from './Wrapper'


const App = () =>
  <Wrapper>
    <Nav />
    <Route exact path="/" component={TrainingPage} />
    <Route exact path="/editor" component={EditorPage} />
  </Wrapper>

export default App
