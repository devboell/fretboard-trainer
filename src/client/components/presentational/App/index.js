import React from 'react'
import { Route } from 'react-router-dom'
import Nav from 'components/presentational/Nav'
import TrainingPage from 'components/presentational/TrainingPage'
import EditorPage from 'components/presentational/EditorPage'
import Wrapper from './Wrapper'


const App = () =>
  <Wrapper>
    <Nav />
    <Route exact path="/" component={TrainingPage} />
    <Route exact path="/editor" component={EditorPage} />
  </Wrapper>

export default App
