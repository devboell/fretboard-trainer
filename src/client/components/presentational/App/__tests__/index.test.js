import React from 'react'

import App from '../index'


it('App component, snapshot', () => {
  const wrapper = shallow(<App />)
  expect(wrapper).toMatchSnapshot()
})
