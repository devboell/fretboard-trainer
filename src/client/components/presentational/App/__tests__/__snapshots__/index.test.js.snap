// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component, snapshot 1`] = `
<Wrapper>
  <Nav />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/editor"
  />
</Wrapper>
`;
